mods/README.md

```diff
@@ -21,321 +21,321 @@ The list should be structured in the following way:
 
 ## 3D_Black
 
-[Fan Covers](https://github.com/VoronDesign/Voron-2/tree/master/mods/3D_Black)  
+[Fan Covers](../mods/3D_Black)  
 
 - Different fan covers for 40mm fans.  
 
 ## Andreq
 
-[Mobius Shaft Jig](https://github.com/VoronDesign/Voron-2/tree/master/mods/andreq/voron2)
+[Mobius Shaft Jig](../mods/andreq/voron2)
 
 - Tool used for grinding flats on a shaft at specific places.
 
 ## Armstr0ng
 
-[Led extrusions mount]((https://github.com/VoronDesign/Voron-2/tree/master/mods/Armstr0ng))
+[Led extrusions mount]((../mods/Armstr0ng))

```

Comment:
Hey @TorinoFermic I tested this link and it seems broken. I think it's the double parenthesis. Mind taking a look?

Reply:
Thanks!